cluster/manifests/node-monitor/daemonset.yaml

```diff
@@ -4,16 +4,14 @@ metadata:
   name: node-monitor
```

Comment:
Then should this daemonset disappear as well?

Reply:
It's just the service that is not used. The daemonset itself is.

The main motivation for removing an unused service is that kube-proxy have to generate iptable rules for service -> pod routing on EVERY node. And when we don't use the service this can be avoided.